File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/api/ppt_template_rendering.py
from typing import Any, Dict, List, Optional
from ..http_client import HttpClient
from ..models import PlusApiResult, PlusPptTemplateRenderForm

class PptTemplateRenderingApi:
    """ppt_template_rendering API client."""
    
    def __init__(self, client: HttpClient):
        self._client = client

    def render(self, body: PlusPptTemplateRenderForm) -> PlusApiResult:
        """Render PPT template"""
        return self._client.post(f"/backend/v3/api/ppt/template/render", json=body)

File: sdkwork-backend-sdk-flutter/lib/src/api/ppt_template_rendering.dart
import '../http/client.dart';
import '../models.dart';

class PptTemplateRenderingApi {
  final HttpClient _client;
  
  PptTemplateRenderingApi(this._client);

  /// Render PPT template
  Future<PlusApiResult?> render(PlusPptTemplateRenderForm body) async {
    final response = await _client.post(ApiPaths.backendPath('/ppt/template/render'), body: body, contentType: 'application/json');
    return response is PlusApiResult ? response : null;
  }
}
